pkg/eval/builtin_special.go

```diff
@@ -331,23 +336,30 @@ func use(fm *Frame, spec string, r diag.Ranger) (*Ns, error) {
 		path := filepath.Clean(dir + "/" + spec)
 		return useFromFile(fm, spec, path, r)
 	}
+
+	// Handle imports of pre-defined modules like `builtin` and `str`.
 	if ns, ok := fm.Evaler.modules[spec]; ok {
 		return ns, nil
 	}
 	if code, ok := fm.Evaler.BundledModules[spec]; ok {
 		return evalModule(fm, spec,
 			parse.Source{Name: "[bundled " + spec + "]", Code: code}, r)
 	}
+
+	// Handle imports relative to the Elvish config lib directories.
```

Comment:
The "config" here is superfluous since config directory and lib directories are now separate concepts.

Reply:
I decided "module search directories." is clearer and consistent with the user-facing documentation.